syntax = "proto3";

message LocationMessage {
    int32 person_id = 1;
    string creation_time = 2;
    string latitude = 3;
    string longitude = 4;
}

message Location {
    bool location = 1;
}

service LocationService {
    rpc create_location(LocationMessage) returns (Location);
}